spyder_kernels/customize/spyderpdb.py

```diff
@@ -188,6 +187,8 @@ def default(self, line):
                     code_ast, capture_last_expression = capture_last_Expr(
                         code_ast, "_spyderpdb_out")
 
+                globals["__spyder_builtins__"] = builtins
```

Comment:
if the user has a local variable called globals, the builting globals is not accessigble anymore. Fix by adding __spyder_builtins__ to access builtins